Docs/Luadoc/Lua.xml

```diff
@@ -968,6 +970,7 @@
 		<Class base='ActorFrame' name='NoteField'>
 			<Function name='DidHoldNote'/>
 			<Function name='DidTapNote'/>
+			<Function name='get_column_actors'/>

```

Comment:
Is there a reason why this call is not `GetColumnActors`? I know we're not always consistent on C++ coding rules here, but this seems strange.